frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "1.0.0",
   "private": false,
   "scripts": {
-    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
+    "start": "react-scripts start",
```

Comment:
Pls set it as previous. 

"start": "DISABLE_ESLINT_PLUGIN=true react-scripts start"